x-pack/plugin/sql/jdbc/src/test/java/org/elasticsearch/xpack/sql/jdbc/TypeConverterTests.java

```diff
@@ -41,9 +44,22 @@ public void testDoubleAsNative() throws Exception {
     }
 
     public void testTimestampAsNative() throws Exception {
-        ZonedDateTime now = ZonedDateTime.now(Clock.tick(Clock.system(ZoneId.of("Z")), Duration.ofMillis(1)));
-        assertThat(convertAsNative(now, EsType.DATETIME), instanceOf(Timestamp.class));
-        assertEquals(now.toInstant().toEpochMilli(), ((Timestamp) convertAsNative(now, EsType.DATETIME)).getTime());
+        ZonedDateTime now = ZonedDateTime.now(Clock.tick(Clock.system(UTC), Duration.ofMillis(1)));
```

Comment:
Maybe extract this into a method - `now(ZoneId)`?

Reply:
I think we used to have a `now` method in SQL somewhere, but if I remember well, it got moved. Searching the ES code, I found `org.elasticsearch.common.time.DateUtils.nowWithMillisResolution()`.